melos.yaml

```diff
@@ -18,6 +13,13 @@ scripts:
     exec: dart format .
     description: Format all Dart packages according to linter settings.
 
+  release:
+    run: bash scripts/release.sh
```

Comment:
The apk files which come out of this are named really badly:

```
app-arm64-v8a-normal-release.apk
app-armeabi-v7a-normal-release.apk
app-normal-release.apk
app-x86_64-normal-release.apk
```

Is there a way to give them more helpful names, like:

```
meli-arm64-v8a-v0.1.0-1-release.apk
```

.. or similar?

Reply:
I've experimented with this here but I end up deep in Gradle errors and I don't know what I'm doing:

https://stackoverflow.com/questions/67186770/flutter-set-version-name-in-apk-filename